src/Engine/ProtoCore/Namespace/ElementResolver.cs

```diff
@@ -1,4 +1,6 @@
+
 ﻿using System.Collections.Generic;
```

Comment:
`Element` over here I guess mainly refers to namespace/class resolution. It was a term given by Luke. I'll just leave it at that.